proto/zigbeealliance/distributedcomplianceledger/model/tx.proto

```diff
@@ -47,6 +47,8 @@ message MsgCreateModel {
   string enhancedSetupFlowTCDigest = 22 [(gogoproto.moretags) = "validate:\"required_if_bit_0_set=EnhancedSetupFlowOptions,omitempty,max=128\""];
   uint32 enhancedSetupFlowTCFileSize = 23 [(gogoproto.moretags) = "validate:\"required_with=EnhancedSetupFlowTCUrl\""];;
   string maintenanceUrl = 24 [(gogoproto.moretags) = "validate:\"required_if_bit_0_set=EnhancedSetupFlowOptions,omitempty,url,startswith=https:,max=256\""];
+  uint32 discoveryCapabilitiesBitmask = 25 [(gogoproto.moretags) = "validate:\"gte=0,lte=14\""];
```

Comment:
It seems we don't do the following validation for `discoveryCapabilitiesBitmask`:
> This field SHALL be populated if CommissioningFallbackUrl is populated

Reply:
The `discoveryCapabilitiesBitmask ` field is always populated (according to spec, it could be 0-14). So, if `commissioningFallbackUrl ` is populated than will be 0(default value) or 1-14